### PR TITLE
Support ARM macs

### PR DESCRIPTION
**Pull request by ineiti**
This modification bundles an xcframework containing fat binaries to support the simulator on ARM Macs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,9 @@ The code is licensed under the MIT license.
 - Christian Grigis, C4DT
 - Laurent Girod, SPRING Lab, EPFL
 
+External contributors:
+- Benjamin Erhart, https://github.com/tladesignz
```
Comment (ineiti) [inline README.md:135]: @tladesignz - if you'd like another mention here, please say so.

File: Makefile [hunk 2]
```diff
@@ -36,3 +36,10 @@ dev:
 dircache:
 	mkdir -p directory-cache
 	wget --output-document - --quiet '$(LATEST_DIRCACHE_URL)' | tar -C directory-cache -zxf -
+
+check-targets:
+	for target in x86_64-apple-ios aarch64-apple-ios-sim aarch64-apple-ios; do \
+		if ! rustup target list | grep installed | grep $$target; then \
+			rustup target add $$target; \
+		fi \
+	done
```
Comment (tharvik) [inline Makefile:45]: that breaks installation not using `rustup` (such as mine and all system-installed rust).

anyway, target support is actively checked by `ios/build_xcf.sh` so we can directly let it fail there

File: .github/workflows/ios-release.yaml [hunk 3]
```diff
@@ -30,6 +30,12 @@ jobs:
           toolchain: stable
           target: aarch64-apple-ios
 
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          target: aarch64-apple-ios-sim
```
Comment (tharvik) [inline .github/workflows/ios-release.yaml:37]: can we merge this block with the previous ones? having multiple target that is

**Review by tharvik — CHANGES_REQUESTED** [review]

**Comment by tharvik** [comment]
FYI, I'm not running macOS so I'm unable to actually check that the script works as intended

File: .github/workflows/ios-release.yaml (hunk 3 above)
Comment (ineiti) [reply .github/workflows/ios-release.yaml]: I tried space separated and comma separated, but this didn't work - any other suggestions?

https://github.com/c4dt/lightarti-rest/actions/runs/3428008325/jobs/5711710484#step:5:50
https://github.com/c4dt/lightarti-rest/actions/runs/3428000679/jobs/5711692270#step:5:50

Comment (ineiti) [reply .github/workflows/ios-release.yaml:37]: According to https://github.com/actions-rs/toolchain it's a string.

**Comment by ineiti** [comment]
> FYI, I'm not running macOS so I'm unable to actually check that the script works as intended

I tested it locally and by running the `ios-release.yaml` workflow.

File: README.md (hunk 1 above)
Comment (tladesignz) [reply README.md:135]: This is fine, no worries.

File: .github/workflows/ios-release.yaml (hunk 3 above)
Comment (tharvik) [reply .github/workflows/ios-release.yaml:37]: hum, no, and I don't see a way to actual expand multiples target [in the code](https://github.com/actions-rs/core/blob/master/src/commands/rustup.ts#L146), nevermind

**Review by tharvik — APPROVED** [review]

**Pull request merged**